frame/staking/src/pallet/mod.rs

```diff
@@ -60,8 +60,12 @@ pub mod pallet {
 
 	use super::*;
 
+	/// The current storage version.
+	const STORAGE_VERSION: StorageVersion = StorageVersion::new(13);
```

Comment:
technically we could have started from 0 as well, but whatever :P

Reply:
Technically yes :). But may be still nice to convey this is the 13th migration. 😉